app/styles/Home.module.css

```diff
@@ -0,0 +1,56 @@
+.container {
```

Comment:
I'm interested in discussing whether we want the default template repo to use CSS or Sass. @lorenyu @sawyerh Any thoughts?

Reply:
Up to you! Switching to Sass in a different PR is ok with me

Reply:
I'll include it in this one!

Reply:
Let's do Sass in a future PR.